### PR TITLE
Support for bridge-networks with netmask smaller than 16 bits

### LINKED ISSUE #3943: Support for bridge-networks with netmask smaller than 16 bits
#For stories, please include the information below:

**User Statement:**

We would like to provide more concurrent users with separate vCH environments with less containers. To be able to do so without complex network gear integration or NSX we would like to be able to create bridge-networks with smaller netmasks (value higher than 16 bits).

**Details:**

**Acceptance Criteria:**
Support for bridge-network range with subnet mask value higher than 16 bits.

#For bug reports, please include the information below:

**VIC version:**

0.8.0

**Deployment details:**

What was the vic-machine create command used to deploy the VCH?

**Steps to reproduce:**

**Actual behavior:**

**Expected behavior:**

**Logs:**

**Additional details as necessary:**

### PR DESCRIPTION
**Pull request by yuyangbj**
Using vic-machine --bridge-network-range, VIC could specify bridge network ip range with netmask smaller than 16 bits; using vic-machine --bridge-network-range, VIC could specify the default bridge network allocation width.

Fixes #3943 #3771
[specific ci=Group23-VIC-Machine-Service]

### CODE REVIEW

File: lib/install/validate/network.go [hunk 1]
```diff
@@ -318,13 +318,8 @@ func (v *Validator) network(op trace.Operation, input *data.Data, conf *config.V
 	// we also need to have the appliance attached to the bridge network to allow
 	// port forwarding
 	conf.AddNetwork(bridgeNet)
-
-	// make sure that the bridge IP pool is large enough for bridge networks
-	err = v.checkBridgeIPRange(input.BridgeIPRange)
```
Comment (wjun) [inline lib/install/validate/network.go]: Any reason to remove this validation? It is used by both create and configure commands. The current code structure put all validations under validate package.

File: cmd/vic-machine/create/create.go [hunk 2]
```diff
@@ -420,6 +429,16 @@ func (c *Create) ProcessBridgeNetwork() error {
 	if err != nil {
 		return cli.NewExitError(fmt.Sprintf("Error parsing bridge network ip range: %s. Range must be in CIDR format, e.g., 172.16.0.0/12", err), 1)
 	}
+	width, err := strconv.Atoi(c.BridgeNetworkWidth)
```
Comment (wjun) [inline cmd/vic-machine/create/create.go]: Better to move validation codes into validator.network() api, so rest apis will share the same validation codes.

File: lib/install/validate/network.go (hunk 1 above)
Comment (yuyangbj) [reply lib/install/validate/network.go]: These codes only serve for create command, we do not support configure for bridge-network-range and bridge-network-width.

File: cmd/vic-machine/create/create.go (hunk 2 above)
Comment (yuyangbj) [reply cmd/vic-machine/create/create.go]: I leave the validation for rest api defined in swagger using minimum and maximum.

File: lib/install/validate/network.go (hunk 1 above)
Comment (wjun) [reply lib/install/validate/network.go]: Then checkBridgeIPRange is not used anymore. If so, please remove this api.

**Pull request merged**